ui.tests/test-module/specs/replaceaction/replacelayout.authoring.spec.js

```diff
@@ -0,0 +1,141 @@
+    // we can use these values to log in
+
+    const pagePath = "/content/forms/af/core-components-it/blank",
+        buttonEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/button",
+        buttonEditPathSelector = "[data-path='" + buttonEditPath + "']",
+        buttonDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.formbutton.split("/").pop();
+
+    const fileInputEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/fileinput",
+        fileInputEditPathSelector = "[data-path='" + fileInputEditPath + "']",
+        fileInputDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.formfileinput.split("/").pop();
+
+    const containerEditPath = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/panelcontainer",
+        containerEditPathSelector = "[data-path='" + containerEditPath + "']",
+        containerDrop = pagePath + afConstants.FORM_EDITOR_FORM_CONTAINER_SUFFIX + "/" + afConstants.components.forms.resourceType.panelcontainer.split("/").pop();
+
+    const containerDataPath = "/content/forms/af/core-components-it/blank/jcr:content/guideContainer/*",
+        componentDataPath = "/content/core-components-examples/library/adaptive-form/button/jcr:content/root/responsivegrid/demo/component/container/*";
+
+    const dropComponent = function (componentEditPathSelector, dataPath) {
+        const responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-path='" + dataPath + "']";
+        cy.selectLayer("Edit");
+        switch (componentEditPathSelector) {
+            case buttonEditPathSelector:
+                cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Button", afConstants.components.forms.resourceType.formbutton);
+                break;
+            case fileInputEditPathSelector:
+                cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form File Attachment", afConstants.components.forms.resourceType.formfileinput);
+                break;
+            case containerEditPathSelector:
+                cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Panel", afConstants.components.forms.resourceType.panelcontainer);
+                break;
+        }
+        cy.get('body').click(0, 0);
+    }
+
+    const testReplaceForFileInput = function (componentEditPathSelector, componentDrop, isSites) {
+        isSites ? dropComponent(componentEditPathSelector, componentDataPath) : dropComponent(componentEditPathSelector, containerDataPath);
+
+        cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + componentEditPathSelector);
+        cy.get("[data-action='replace']").should("not.exist");
+        cy.deleteComponentByPath(componentDrop);
+    }
+
+    const testComponentReplaceBehaviour = function (componentEditPathSelector, componentDrop, isSites) {
```

Comment:
can't see any test case which sends isSites argument . If we don't need it , we should remove